kvpress/presses/adakv_press.py

```diff
@@ -34,8 +34,16 @@ def compression_ratio(self):
     def compression_ratio(self, value):
         self.press.compression_ratio = value
 
+    @property
```

Comment:
Any changes to the presses implementation should be removed in this PR. If you want to discuss potential core changes, please open an issue; we can discuss there.